packages/saved-views-react/src/translation/displayStyleExtractor.ts

```diff
@@ -770,6 +770,11 @@ export const extractDisplayStyle3d = (data: object) => {
     styles = (data as LegacySavedView3d).displayStyleProps.jsonProperties?.styles;
     applyExtraction(styles, output, displayStyle3dLegacyMapping);
   }
+  const range = output?.thematic?.range;
+    if (range && Array.isArray(range) && range.length === 0) {
+      // Range is optional, so delete it if it's empty
+      delete output.thematic.range;
```

Comment:
Using `delete` operator in JavaScript really hurts performance in V8. It would be better to assign `undefined` value to property.

Reply:
I wasn't sure if setting it to undefined would cause problems (as it's done that before, for some things), but I think since range is optional, it should be okay...
It's changed now.